packages/components-react/icon-react/src/icon.tsx

```diff
@@ -2,6 +2,8 @@ import { clsx } from 'clsx';
 import type { HTMLAttributes } from 'react';
 import { forwardRef } from 'react';
 
+export type IconSize = 'em' | 'ex';
```

Comment:
Ik zou ook een expliciete optie toevoegen voor whatever de default optie wordt, zoals `ic` of `lh`.